job_definitions/index_briefs.yml

```diff
@@ -0,0 +1,44 @@
+{% for environment in ['preview'] %}
+- job:
+    name: "index-briefs-{{ environment }}"
+    display-name: "Index briefs - {{ environment }}"
+    project-type: freestyle
+    description: "This imports all briefs (opportunities) in the current API database into the current Elasticsearch index."
+    parameters:
+      - string:
+          name: INDEX
+          default: {{ search_config[environment].default_index }}
+          description: "Index name or alias to use (eg 'briefs-digital-outcomes-and-specialists-2')"
+      - string:
+          name: FRAMEWORKS
+          default: {{ search_config[environment].frameworks }}
+          description: >
+            Comma-separated list of framework slugs that should be indexed
+            (eg 'digital-outcomes-and-specialists-1,digital-outcomes-and-specialists-2'). If
+            no frameworks are specified then all currently published briefs will be indexed.
+      - string:
+          name: CREATE_WITH_MAPPING
+          default:
+          description: >
+            Specify a mapping to create the index if it doesn't exist. If it does
+            exist, the mapping for the index will be updated. Don't specify this
+            option when running from a batch job, as both creating indexes and updating
```

Comment:
`batch job` should maybe say regular/automated job? Running this at all is presumably a batch job in that it indexes a batch of services (unless my nomenclature is off).

If this option should never be provided to the job at all, then we should just not offer it at all?

Reply:
Yeah, I had trouble with the wording here. It is intended that you can use the option when running manually, and Jenkins is supposed to be the easy way to do that (a fact I forgot when writing the docopt in the script).

I think that technically 'batch' refers to the fact that the _jobs_ are batched up to run (i.e. automatically), but seeing as it's definitely confusing, let's try another word instead. Of course, even the manual jobs are automated on some level...

How about "This option should only be used when running under manual control (i.e. never from an automated/timed job), as...".

"Timed" is the word that we use in the "triggers" section of the job definition, anyway.